persistence/postgres-persistence/src/main/java/com/netflix/conductor/postgres/dao/PostgresIndexDAO.java

```diff
@@ -299,4 +299,9 @@ public long getWorkflowCount(String query, String freeText) {
         logger.info("getWorkflowCount is not supported for postgres indexing");
         return 0;
     }
+
+    @Override
+    public SearchResult<WorkflowSummary> getSummaries(SearchResult<String> searchResultIds) {
+        throw new UnsupportedOperationException("Not implemented yet.");
+    }
```

Comment:
Based on previous methods I will be standardize this class to one form (fail-fast with UnsupportedOperationException or not fail-fast with logging error and returning some negative value - 0, null or false)

Reply:
Resolved.